admin/bootstrap.py

```diff
@@ -63,10 +63,12 @@ def run_command(command: List[str]) -> Iterator[bytes]:
 
 
 def is_tails() -> bool:
```

Comment:
I should've checked earlier because I misled you yesterday :( Tails has a non-standard `/etc/os-release`, it looks like:
```
TAILS_PRODUCT_NAME="Tails"
TAILS_VERSION_ID="5.0"
TAILS_DISTRIBUTION="unstable"
```

Since this function is just trying to detect whether we're on tails or not, it could be as simple as `return "TAILS_PRODUCT_NAME" in f.read()`.

That should work across recent Tails 4.x and 5.x, so we don't even need the "dirty hack" anymore.